reverse-proxy/api/index.html

```diff
@@ -3,15 +3,15 @@
   <head>
     <title>YARP Documentation</title>
     <link rel="shortcut icon" href="//www.microsoft.com/favicon.ico?v2" />
-    <meta http-equiv="refresh" content="2;url=https://dotnet.github.io/yarp/api/index.html" />
+    <meta http-equiv="refresh" content="2;url=https://learn.microsoft.com/search/?terms=YARP%20API%20Reference&category=Reference" />
     <meta charset="UTF-8">
     <script type="text/javascript">
-      location.href = 'https://dotnet.github.io/yarp/api/index.html';
+      location.href = 'https://learn.microsoft.com/search/?terms=YARP%20API%20Reference&category=Reference';
     </script>
   </head>
   <body>
     <p style="font-family:'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;font-size:18px;text-align:center">
-      YARP documentation has been moved to <a href="https://dotnet.github.io/yarp/api/index.html">https://dotnet.github.io/yarp/api/index.html</a>.
+      YARP documentation has been moved to <a href="https://learn.microsoft.com/search/?terms=YARP%20API%20Reference&category=Reference">https://learn.microsoft.com/search/?terms=YARP%20API%20Reference&category=Reference</a>.
```

Comment:
For the API one, I expect it to be fairly stable so maybe we should just link to https://learn.microsoft.com/en-us/dotnet/api/yarp.reverseproxy or something here?

Reply:
This redirect only works for the index site
https://microsoft.github.io/reverse-proxy/api/index.html

Links to any specific API like
https://microsoft.github.io/reverse-proxy/api/Microsoft.AspNetCore.Builder.AppBuilderDelegationExtensions.html
are and will remain broken.